libbeat/processors/add_process_metadata/gosigar_cid_provider.go

```diff
@@ -104,7 +107,7 @@ func (p gosigarCidProvider) getCid(cgroups map[string]string) string {
 	for _, path := range cgroups {
 		for _, prefix := range p.cgroupPrefixes {
 			if strings.HasPrefix(path, prefix) {
-				return filepath.Base(path)
+				return filepath.Base(strings.TrimSuffix(path, "/kube-proxy"))
```

Comment:
I propose we remove this to split concerns in different PRs to make the other part move forward.

Reply:
Do you need me to do something for that?

Reply:
in my env i went through all cgroups, there is only this same kube-proxy that differs

Reply:
yes, could you please add a commit undoing the change to this line? Or just accept this suggestion :innocent: :
```suggestion
				return filepath.Base(path)
```